emscripten/main.cpp

```diff
@@ -33,22 +33,20 @@ uint8_t *loadImage(std::string filename, int width, int height) {
     return (uint8_t *) img;
 }
 
-EM_JS(emscripten::EM_VAL, get_video_stream, (), {
+EM_JS(int, get_video_stream, (), {
 var video = document.getElementById('video');
 var canvas_process = document.createElement('canvas');
 var context_process = canvas_process.getContext('2d');
 context_process.fillStyle = 'black';
-context_process.fillRect(0, 0, 640, 480);
-context_process.drawImage(video, 0, 0, 640, 480);
-var imageData = context_process.getImageData(0, 0, 640, 480);
+(context_process.fillRect(0, 0, 640, 480));
```

Comment:
note the parenthesis... i was getting strange issues by the acorn closure compiler...